utils/post_utils.jsx

```diff
@@ -323,20 +313,20 @@ export function makeCreateAriaLabelForPost() {
     const getDisplayName = makeGetDisplayName();
 
     return createSelector(
-        (state, post) => post,
+        (_state, post) => post,
```

Comment:
Nit: `state` was fine too. 
For large ones, The less changes in a PR, the less things to review and easier history when we look for changes to file.